test/tests/mesh/node_list_from_side_list/node_list_from_side_list.i

```diff
@@ -1,9 +1,16 @@
 [Mesh]
-  type = GeneratedMesh
-  dim = 2
-  nx = 10
-  ny = 10
-  construct_node_list_from_side_list = false
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 10
+    ny = 10
+  []
+  [rename]
+    type = RenameBoundaryGenerator
+    input = gen
+    old_boundary_name = 'left'
+    new_boundary_name = 'renamed_left'
```

Comment:
```suggestion
    old_boundary = 'left'
    new_boundary = 'renamed_left'
```

Reply:
I take it this suggestion is related to the new changes?

Reply:
Yep. Takes mixed id/names for {old/new}_boundary